Resources/Private/Layouts/Default.html

```diff
@@ -7,7 +7,19 @@
 		<div id="typo3-docheader">
 			<div class="typo3-docheader-functions">
 				<div class="left">
-					<solr:backend.menu.SiteSelectorMenu name="site" />
+
+					<solr:backend.sites>
+						<f:if condition="{hasSites}">
+							<f:then>
+								<solr:backend.menu.SiteSelectorMenu name="site" />
+							</f:then>
+							<f:else>
+								No solr connections initialized!
```

Comment:
When talking about the Solr server, please use uppercase S "Solr". When talking about the extension, it's EXT:solr with lowercase s.